src/common/reducers/handlers/cards.js

```diff
@@ -145,4 +121,25 @@ function createCardFromProps(props) {
   return card;
 }
 
+// Saves a card, either as a new card or replacing an existing card.
+function saveCard(state, card) {
+  // Is there already a card with the same ID (i.e. we're currently editing it)
+  // or that is identical to the saved card (i.e. we're replacing it with a card with the same name)?
+  const existingCard = state.cards.find(c => c.id === card.id || areIdenticalCards(c, card));
```

Comment:
We want to check if it is an existing card without using the `id` field. We don't want random people using the `id` of a core-set card to replace the core set card.